radio/src/storage/conversions/yaml/yaml_datastructs.cpp

```diff
@@ -54,6 +54,8 @@ namespace yaml_conv_220 {
   #include "yaml_datastructs_zorro.cpp"
  #elif defined(RADIO_T8)
   #include "yaml_datastructs_t8.cpp"
+ #elif defined(RADIO_COMMANDO8)
+  #include "yaml_datastructs_commando8.cpp"  
```

Comment:
Please remove all changes in `storage/conversions/*`: you would only need them to support a pre-existing binary storage. As EdgeTx does not support binary storage for new radios, it should not be needed.